From e1483e8a2109fb8b70e20535c32c69911d878ff4 Mon Sep 17 00:00:00 2001
From: CHINTU <harrypetersun@outlook.com>
Date: Fri, 30 Jul 2021 13:03:29 +0000
Subject: [PATCH] Add exact calculator

---
 config/packages.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/config/packages.mk b/config/packages.mk
index 1d90bdf..5ef2ea6 100644
--- a/config/packages.mk
+++ b/config/packages.mk
@@ -1,3 +1,7 @@
+# LineageOS apps
+PRODUCT_PACKAGES += \
+    ExactCalculator
+
 # Packages
 PRODUCT_PACKAGES += \
     Corvus-Themes \
